src/internal/executer.ts

```diff
@@ -21,7 +21,7 @@ export class RequestExecuter {
     await this.systemInit;
     const token = await this.tokenManager.token(this.config.auth);
     return this.requestAdapter.execute(
-      this.getUrl(),
+      this.getURI(request.queryParams || []),
       {
         headers: { Authorization: `Bearer ${token}` },
         body: request.body || {},
```

Comment:
Could `|| {}` be removed, what do you think?

Reply:
done!